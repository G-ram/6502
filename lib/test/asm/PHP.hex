78 08 f8 28 
