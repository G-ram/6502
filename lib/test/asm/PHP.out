A: 00
X: 00
Y: 00
P: 00010100
S: 20
PC: 604
100 : 00
101 : 00
